euphonic/powder.py

```diff
@@ -366,20 +366,20 @@ def _get_qpts_sphere(npts: int,
 
     if sampling == 'golden':
```

Comment:
Are you 3.10+? Is this a place for `match`?

Reply:
Looks like an ideal place for it; done!